test-suite/src/values.rs

```diff
@@ -87,8 +85,33 @@ test_case!(values, async move {
             }
             .into()),
         ),
+        (
+            "CREATE TABLE TableFromValues AS VALUES (1, 'a', True, Null, Null), (2, 'b', False, 3, Null)",
+            Ok(Payload::Create),
+        ),
+        (
+            "SELECT * FROM TableFromValues",
+            Ok(select_with_null!(
+                column1 | column2         | column3    | column4 | column5;
+                I64(1)    Str("a".into())   Bool(true)   Null      Null   ;
+                I64(2)    Str("b".into())   Bool(false)  I64(3)    Null
+            )),
+        ),
     ];
     for (sql, expected) in test_cases {
         test!(expected, sql);
     }
 });
+
+test_case!(type_match, async {
+    type_match!(
+        &[
+            DataType::Int,
+            DataType::Text,
+            DataType::Boolean,
+            DataType::Int,
+            DataType::Text,
+        ],
+        "SELECT * FROM TableFromValues"
+    );
+});
```

Comment:
It looks that `type_match` is not registered in integration tests, so it is not being tested.

Reply:
Thank you for pointing out!
I just fixed that part but realized that it does not match schema but data.
Is there any test to match schema?

Reply:
I substituted with `show columns from~` test !